packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -139,18 +139,20 @@ class DisplayOptions {
 
 class DeepLinksController extends DisposableController {
   DeepLinksController() {
-    selectedVariantIndex.addListener(_handleSelectedVariantIndexChanged);
+    selectedAndroidVariantIndex
+        .addListener(_handleSelectedAndroidVariantIndexChanged);
```

Comment:
do we also need to add a listener for the ios variant index here?

Reply:
Yes, let me add a todo here

this pr just added ios build option, will add the listener  when loading ios links in next PR,